src/data/cosmos.js

```diff
@@ -123,6 +123,14 @@ async function getFormDataByUserIdWithNewAuthToken(userId) {
   return item;
 }
 
+async function saveFormData(authToken, formData) {
+  const item = await getFormDataByAuthToken(authToken);
+  item.formData = formData;
+  item.confirmationNumber = uuidv4();
```

Comment:
Yeah, we need to decide what confirmation numbers look like. A uuid is maybe too confusing. Anyway, this is fine for now, but we should circle back with the designers & EDD.